File: chpt 3 assign translated to java/NestedLoops.java
public class NestedLoops {
    public static void main(String[] args) {
        
        for (int row = 0; row < 10; row++) {
            for (int column = 0; column <= row; column++) {
                System.out.print("*");
            }
            System.out.println();
        }

        System.out.println();  

        for (int row = 10; row > 0; row--) {
            for (int column = 0; column < row; column++) {
                System.out.print("*");
            }
            System.out.println();
        }

        System.out.println();  

        for (int row = 0; row < 10; row++) {
            for (int column = 0; column < 10 - row; column++) {
                System.out.print(" ");
            }
            for (int asterik = 0; asterik <= row; asterik++) {
                System.out.print("*");
	    }
            System.out.println();
        }

        System.out.println();  

        for (int row = 10; row > 0; row--) {
            for (int column = 0; column < 10 - row; column++) {
                System.out.print(" ");
            }
            for (int asterik = 0; asterik < row; asterik++) {
                System.out.print("*");
            }
            System.out.println();
        }
    }
}

File: python chpt 3 assignments/nestedloops.py
for row in range(10):
    for column in range(row+1):
        print('*', end='')
    print()

for row in range(10, 0, -1):
    for column in range(row):
        print('*', end='')
    print()

print()  

for row in range(10):
    for column in range(10-row):
        print(' ', end='')
    for asterik in range(row+1):
        print('*', end='')
    print()

print()  

for row in range(10, 0, -1):
    for column in range(10-row):
        print(' ', end='')
    for asterik in range(row):
        print('*', end='')
    print()

